Shared/UserAgent.swift

```diff
@@ -45,59 +45,51 @@ open class UserAgent {
                 return firefoxUA
             }
         }
-
         return nil
     }
 
+    private static var systemDefaultUA: String = ""
     /**
      * This will typically return quickly, but can require creation of a UIWebView.
      * As a result, it must be called on the UI thread.
      */
     public static func defaultUserAgent() -> String {
         assert(Thread.current.isMainThread, "This method must be called on the main thread.")
-
         if let firefoxUA = UserAgent.cachedUserAgent(checkiOSVersion: true) {
             return firefoxUA
         }
-
-        let webView = UIWebView()
-
+        // Reset UA
```

Comment:
All of these changes look a bit suspicious. We should pull down FF's latest `Shared/UserAgent.swift` code, and use that. If that is what this is, great, if not, let's get rid of it.